tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1273,6 +1274,52 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		framework.Logf("nodePort tidbcluster tidb service NodePort haven't changed after update")
 	})
 
+	ginkgo.It("[Feature: CDC]", func() {
+		ginkgo.By("Creating cdc cluster")
+		fromTc := fixture.GetTidbCluster(ns, "cdc-source", utilimage.TiDBNightly)
+		fromTc.Spec.PD.Replicas = 3
+		fromTc.Spec.TiKV.Replicas = 3
+		fromTc.Spec.TiDB.Replicas = 2
+		fromTc.Spec.TiCDC = &v1alpha1.TiCDCSpec{
+			BaseImage: "pingcap/ticdc",
+			Replicas:  3,
+		}
+		err := genericCli.Create(context.TODO(), fromTc)
+		framework.ExpectNoError(err, "Expected TiDB cluster created")
+		err = oa.WaitForTidbClusterReady(fromTc, 30*time.Minute, 15*time.Second)
+		framework.ExpectNoError(err, "Expected TiDB cluster ready")
+
+		ginkgo.By("Creating cdc-sink cluster")
+		toTc := fixture.GetTidbCluster(ns, "cdc-sink", utilimage.TiDBNightly)
+		toTc.Spec.PD.Replicas = 1
+		toTc.Spec.TiKV.Replicas = 1
+		toTc.Spec.TiDB.Replicas = 1
+		err = genericCli.Create(context.TODO(), toTc)
+		framework.ExpectNoError(err, "Expected TiDB cluster created")
+		err = oa.WaitForTidbClusterReady(toTc, 30*time.Minute, 15*time.Second)
+		framework.ExpectNoError(err, "Expected TiDB cluster ready")
+
+		ginkgo.By("Creating change feed task")
+		fromTCName := fromTc.Name
+		toTCName := toTc.Name
+		changeFeedCMD := fmt.Sprintf("/cdc cli changefeed create "+
+			"--sink-uri=\"tidb://root:@%s:4000/\" --pd=http://%s:2379",
+			controller.TiDBMemberName(toTCName), controller.PDMemberName(fromTCName))
+		cmd := fmt.Sprintf("kubectl exec -n %s %s-0 -- %s",
+			ns, controller.TiCDCMemberName(fromTCName), changeFeedCMD)
+		data, err := exec.Command("sh", "-c", cmd).CombinedOutput()
```

Comment:
```suggestion
		changeFeedCMD := fmt.Sprintf("/cdc cli changefeed create "+
			"--sink-uri=\"tidb://root:@%s:4000/\" --pd=http://%s:2379",
			controller.TiDBMemberName(toTCName), controller.PDMemberName(fromTCName))
		cmd := fmt.Sprintf("kubectl exec -n %s %s-0 -- %s",
			ns, controller.TiCDCMemberName(fromTCName), changeFeedCMD)
		data, err := framework.RunKubectl("exec", "-n", ns, controller.TiCDCMemberName(fromTCName) + "-0", "--", "/cdc", "cli", "changefeed", "create", fmt.Sprintf("--sink-uri=tidb://root:@%s:4000/", controller.TiDBMemberName(toTCName)), fmt.Sprintf("--pd=http://%s:2379", controller.PDMemberName(fromTCName))
```

`framework.RunKubectl` is a convenient wrapper to exec raw kubectl command. With it, we don't need to care about underlying details.

Reply:
cooool.